core/src/main/java/io/ddf/content/ViewHandler.java

```diff
@@ -128,7 +128,7 @@ public DDF removeColumns(List<String> columnNames) throws DDFException {
     }
     List<String> columns = this.getDDF().getColumnNames();
```

Comment:
This is not different from `List<String> currentColumnNames = this.getDDF().getColumnNames();`, why declare again?